src/app/documentation/demos/code/code-highlight-snippet.ts

```diff
@@ -6,15 +6,16 @@
 import {Component} from "@angular/core";
 
 const HTML_EXAMPLE = `
+<!-- "&#123;" left curly brace / "&#125;" right curly brace -->
 <pre>
     <code clr-code-highlight="language-css">
-        .some-component {
+        .some-component &#123;
```

Comment:
I am not exactly sure how you are getting that error. We don't get that error. Also the `new-website` branch is for internal use only. Its used for publishing our website. Why are you using that branch?